infrastructure/azure.tf

```diff
@@ -297,41 +323,41 @@ resource "azurerm_storage_blob" "azurerm_message_blob" {
 ## DATABASE
 
 resource "azurerm_cosmosdb_account" "azurerm_cosmosdb" {
-    name                = "${var.azurerm_cosmosdb}"
-    location            = "${azurerm_resource_group.azurerm_resource_group.location}"
-    resource_group_name = "${azurerm_resource_group.azurerm_resource_group.name}"
-
-    # Possible values are GlobalDocumentDB and MongoDB
-    kind = "GlobalDocumentDB"
-
-    # Required - can be only set to Standard
-    offer_type          = "Standard"
-
-    # Can be either BoundedStaleness, Eventual, Session or Strong
-    # see https://docs.microsoft.com/en-us/azure/cosmos-db/consistency-levels
-    # Note: with the default BoundedStaleness settings CosmosDB cannot perform failover / replication:
-    #   Operations (max_staleness): for a single region the maximum operations lag must be between 10 and 1 000 000
-    #               for the multi region, it will be between 100 000 and 1 000 000
-    #   Time (max_interval_in_seconds): the maximum lag must be between 5 seconds and 1 day for either single or multi-regions
-    consistency_policy {
-        consistency_level = "Session"
-    }
+  name                = "${var.azurerm_cosmosdb}"
+  location            = "${azurerm_resource_group.azurerm_resource_group.location}"
+  resource_group_name = "${azurerm_resource_group.azurerm_resource_group.name}"
+
+  # Possible values are GlobalDocumentDB and MongoDB
+  kind = "GlobalDocumentDB"
+
+  # Required - can be only set to Standard
+  offer_type = "Standard"
+
+  # Can be either BoundedStaleness, Eventual, Session or Strong
+  # see https://docs.microsoft.com/en-us/azure/cosmos-db/consistency-levels
+  # Note: with the default BoundedStaleness settings CosmosDB cannot perform failover / replication:
+  #   Operations (max_staleness): for a single region the maximum operations lag must be between 10 and 1 000 000
+  #               for the multi region, it will be between 100 000 and 1 000 000
+  #   Time (max_interval_in_seconds): the maximum lag must be between 5 seconds and 1 day for either single or multi-regions
+  consistency_policy {
+    consistency_level = "Session"
+  }
 
-    failover_policy {
-        location = "${var.cosmosdb_failover_location}"
-        priority = 0
-    }
+  failover_policy {
+    location = "${var.cosmosdb_failover_location}"
+    priority = 0
+  }
 
-    ip_range_filter = "${azurerm_function_app.azurerm_function_app.outbound_ip_addresses}"
+  # ip_range_filter = "${azurerm_function_app.azurerm_function_app.outbound_ip_addresses}"
```

Comment:
not needed anymore?

Reply:
it does not work because of dependency cycle